src/main/java/org.wso2.apim.monetization/impl/model/MonetizedSubscription.java

```diff
@@ -0,0 +1,93 @@
+/*
+ *  Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.apim.monetization.impl.model;
+
+/**
+ * This class is to define the properties of monetized subscription
+ **/
+public class MonetizedSubscription {
+
+    private int id;
+    private int subscribedApiId;
+    private int subscribedApplicationId;
+    private int tenantId;
+    private String subscriptionId;
+    private int sharedCustomerId;
+
+    public int getId() {
+
```

Comment:
Please remove unwanted new lines in each file.

Reply:
fixed in 49ded003cbdb2832bd0e8845951ac7af9831f3cb